### DISCUSSION TITLE
Support for plotly

### DISCUSSION THREAD

**Discussion by dove88** [posit-dev/positron#3981] [Q&A] [answered]
Does the positron plot view tab in jupyter support python plotly? I even tried the inline chart (plotly) with  jupyter but did not succeed

**Comment by juliasilge** [comment, marked as answer]
I think you've run into the same problem as this: https://github.com/posit-dev/positron/issues/3972

We've got a fix in a PR and it should land in a release build very soon. Thank you for the report! 🙌

**Reply by dove88** [reply]
It's ok now, I found the solution for it, thank you for the positron IDE, I am enjoying data analysis with it, thank you
